app_dart/lib/src/model/luci/buildbucket.dart

```diff
@@ -392,6 +394,16 @@ class ScheduleBuildRequest extends JsonBody {
   @TagsConverter()
   final Map<String, List<String>> tags;
 
+  /// Overrides default dimensions defined by builder config or template build.
+  ///
+  /// A set of entries with the same key defines a new or replaces an existing
+  /// dimension with the same key.
+  final List<RequestedDimension> dimensions;
+
+  // If not zero, overrides swarming task priority.
+  // See also Build.infra.swarming.priority.
+  final int priority;
```

Comment:
Should we have the default 30 here? Ideally we should not change this as changing priorities can potentially make all the builders perform worst than without changing the default priority.

Reply:
We can leave this field unset if there's no need to overwrite the default.